website/project/model.py

```diff
@@ -2749,6 +2749,8 @@ def embargo_registration(self, user, end_date, for_existing_registration=False):
         )
         # Embargo record needs to be saved to ensure the forward reference Node->Embargo
         self.embargo = embargo
+        if self.is_public:
+            self.set_privacy('private', Auth(user))
```

Comment:
I don't have strong feelings about it, but I would guess it would be more intuitive if a log were not added, since the action being performed is "embargo initiated". This also makes the logging consistent between public/private embargoing.